provider-ci/internal/pkg/templates/bridged-provider/Makefile

```
@@ -138,7 +138,7 @@ tfgen: install_plugins upstream#{{ if .Config.docsCmd }}# docs#{{ end }}#
 
 upstream:
 ifneq ("$(wildcard upstream)","")
-	@$(SHELL) ./scripts/upstream.sh "$@" apply
+	scripts/upstream.sh "$@" apply
```

I would rather we called `scripts/upstream.sh` as a script, instead of as an executable. If you don't want to use `$(SHELL)`, we can specify bash directly.

```suggestion
	bash ./scripts/upstream.sh "$@" apply
```

---

What's the motivation for running it in a subshell?

---

AFAIK it will be run in a sub shell regardless. I would prefer we invoke it via the executor to avoid worrying about executability.

---

Executability is what the shebang is for (`#!/usr/bin/env bash`)

[Wikipedia Reference](https://en.wikipedia.org/wiki/Shebang_%28Unix%29)